domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -182,12 +174,11 @@ class CameraXCameraUseCase @Inject constructor(
 
     private fun getZoomState(): ZoomState? = camera?.cameraInfo?.zoomState?.value
 
-    // flips the camera to the designated lensFacing direction
-    override suspend fun flipCamera(isFrontFacing: Boolean) {
-        this.isFrontFacing = isFrontFacing
-        rebindUseCases(
-
+    override suspend fun setLensFacing(lensFacing: CameraUseCase.LensFacing) {
+        _config.emit(
```

Comment:
If performing an action that needs to rebind use cases, we should emit after rebinding since `rebindUseCases` may throw an exception. This will probably make more sense once we have the prescriptive config since right now the config here is used in `rebindUseCase` and is acting as both the prescriptive and descriptive config.